src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -840,13 +843,11 @@ julia> isfulldimensional(p)
 true
 ```
 """
-project_full(P::Polyhedron) = Polyhedron(Polymake.polytope.project_full(pm_object(P)))
-
-
+project_full(P::Polyhedron{T}) where T<:scalar_types = Polyhedron{T}(Polymake.polytope.project_full(pm_object(P)))
 
 @doc Markdown.doc"""
 
-    gelfand_tsetlin(lambda::AbstractVector)
+    gelfand_tsetlin([::Type{T} = fmpq,] lambda::AbstractVector)
```

Comment:
```suggestion
    gelfand_tsetlin(lambda::AbstractVector)
```
Gelfand-Tsetlin polytope should stay with rational coordinates only, see https://github.com/oscar-system/Oscar.jl/pull/1022#issuecomment-1023041708.